applications/triwild/triwild_test_config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "test_directory": "unit_test",
+    "tests": [],
+    "release_only_tests": [
+        "triwild_siggraph_icon_out.json"
```

Comment:
Is this slow? If it doesn't take more than a minute, I'd vote for running this test in debug and release.